maserol/core.py

```diff
@@ -179,18 +177,25 @@ def optimizeLoss(
     jac_sparsity[*idx, *idx] = rcp_ab_block
     jac_sparsity = np.moveaxis(jac_sparsity, (4, 5), (1, 4))
     jac_sparsity = jac_sparsity.reshape(n_samp * n_rcp * n_ag, n_samp * n_ab * n_ag)
-    jac_sparsity = np.pad(jac_sparsity, ((0, 0), (0, 1)), constant_values=1) # cheeky scaling factor
     jac_sparsity = coo_matrix(jac_sparsity)
 
     print("")
+    inner_opt_kwargs = inner_opt_kwargs or {}
+    default_inner_opt_kwargs = {
```

Comment:
Do you need `default_inner_opt_kwargs` long-term? If not I'd remove.

Reply:
Probably not. Removed.